lib/pages/task_list.dart

```diff
@@ -17,90 +19,69 @@ class _TaskListState extends State<TaskList> {
           ),
-          drawer: Drawer(
-              backgroundColor: bgColor,
-              child: ListView(
-                children: [
-                  const UserAccountsDrawerHeader(
-                    margin: EdgeInsets.all(4),
-                    accountName:
-                        Text('Task Manager', style: TextStyle(fontSize: 18)),
-                    accountEmail:
-                        Text('Options', style: TextStyle(fontSize: 18)),
-                    decoration: BoxDecoration(
-                        color: Color.fromARGB(255, 129, 128, 128)),
-                    currentAccountPicture: CircleAvatar(
-                      backgroundColor: Colors.black,
-                      child: Icon(Icons.person, size: 40, color: Colors.white),
-                    ),
-                    arrowColor: Colors.black,
-                  ),
-                  Padding(
-                    padding: const EdgeInsets.fromLTRB(4, 6, 4, 4),
-                    child: Card(
-                      color: const Color.fromARGB(255, 85, 84, 84),
-                      child: GestureDetector(
-                        onTap: () => context.push('/completed_tasks'),
-                        child: const Padding(
-                          padding: EdgeInsets.all(4.0),
-                          child: ListTile(
-                            leading: Icon(Icons.task_alt, color: Colors.white),
-                            title: Text('View Completed Tasks',
-                                style: TextStyle(
-                                    color: Colors.white, fontSize: 18)),
-                          ),
-                        ),
-                      ),
-                    ),
-                  ),
-                ],
-              )),
           body: ListView.builder(
             itemCount: tasks.length,
             itemBuilder: (context, index) {
               return tasks[index];
             },
           ),
-          floatingActionButton: FloatingActionButton(
-            key: const Key('taskAdderFloatingButton'),
-            onPressed: () {
-              context.push('/addtask');
-            },
-            backgroundColor: Colors.black,
-            child: Icon(Icons.add, color: iconColor),
-          ),
         );
       },
     );
   }
+
+  void _showEditPanel(BuildContext context, Task task) {
+    showModalBottomSheet(
+        context: context,
+        builder: (context) {
+          return Container(
+            padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 60),
+            color: bgColor,
+            child: EditTask(
+                task: task,
+                onTaskUpdated: (updatedTask) {
+                  context
+                      .read<TaskCubit>()
+                      .updateTask(updatedTask: updatedTask);
+                }),
+          );
+        });
+  }
+
+  void _onTapCompleteIcon(Task task) {
+    context.read<TaskCubit>().completeTask(task: task);
+  }
```

Comment:
Tip for the future: If you like them, consider arrow functions. This could probably be a one-liner. Please resolve this comment.